1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,4 +1,45 @@
-function setAlarm() {}
+// Get DOM elements
+const timeLeft = document.querySelector("#timeRemaining");
```

Comment:
Comment: I see you choose not to use .getElementsById but .querySelector instead. Should work too.